### ISSUE TITLE
Release process?

### ISSUE THREAD

**Issue by daltonbohning** [hpc/mpifileutils#402]
As part of integrating mpifileutils into our DAOS CI, it is preferred that we build with stable releases instead of directly from `master`. But since we are and will be contributing new changes to mpifileutils, I'm not sure how feasible it is to only build from stable releases. E.g. a release for each DAOS-related change is probably overkill and too much effort on the part of those who manage mpifileutils.

But, to be sure, how difficult and time-consuming is it to get a new mpifileutils release?

**Comment by adammoody** [comment]
@daltonbohning , releases so far have been largely feature-based, with minor bug fix releases coming out soon after to address any new critical bugs.  We have aimed to make major releases at nominal intervals of about 6 months, but we've allowed that to slip when planned features were behind schedule.

The release process itself is not too onerous.  We have documented the steps here:
https://github.com/hpc/mpifileutils/wiki/Release-Process
